srv/GetSpawnPoses.srv

```diff
@@ -0,0 +1,10 @@
+# Get predefined spawn poses which is convenient to avoid spawning in invalid spaces.
+# This is an optional simulation feature: check status_message if you receive an empty spawn_poses vector,
+# to determine whether this feature is not supported by your simulator, or spawn poses are simply not defined.
+
```

Comment:
Does the request need to specify what entity you're getting the spawn poses for?

It could be an optional string payload (of the entity name), but if omitted/blank it could return spawn poses?

I'm thinking for example that robots and tabletop objects would have drastically different spawn poses. Or even different types of robots in the scene -- UAVs vs. mobile bases, etc.

A more ambitious, but probably not necessary, idea is to define named groups of spawn poses. So for example, a group named `"drone_spawn_locations"` could contain a fixed set of locations where you can spawn any drone, etc.

Reply:
I changed the semantics to GetNamedPoses and introduced tags following another suggestion. This should make it possible for the user to determine drone spawn points through checking tags "spawn" and "crazyflie", "turtlebot" etc. Take a look and let me know.